src/main/java/com/swisscom/ais/client/model/VisibleSignatureDefinition.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Swisscom Trust Services (Schweiz) AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.swisscom.ais.client.model;
+
+import java.io.File;
+
+public class VisibleSignatureDefinition {
+
+    public int x;
```

Comment:
Since this is a public class, all these attributes should be private and the class should provide getters and setters for them. Also, a no-args constructor would help with using the setters instead of the full-args constructor.